app/src/main/kotlin/com/mapzen/erasermap/model/AndroidAppSettings.kt

```diff
@@ -3,16 +3,17 @@ package com.mapzen.erasermap.model
 import android.content.Context
 import android.location.Location
 import android.preference.PreferenceManager
+import com.mapzen.android.MapzenMap
 import com.mapzen.erasermap.EraserMapApplication
 import com.mapzen.erasermap.R
 import com.mapzen.pelias.SavedSearch
-import com.mapzen.tangram.DebugFlags
-import com.mapzen.tangram.Tangram
+import com.mapzen.tangram.MapController
```

Comment:
This is actually still used for the debug flags. Will be removed in an upcoming PR though